CMake/Dependencies/libsrtp-CMakeLists.txt

```diff
@@ -37,10 +35,6 @@ else()
   set(LIBSRTP_SHARED_LIBS ON)
 endif()
 
-if (NOT WIN32)
```

Comment:
Not entirely sure why this section is needed, and I didn't see an explanation in the commit. Could you elaborate on this piece?

Reply:
This was a patch that I deleted after verifying the version of the library I'm upgrading to already has those fixes.

Reply:
In short I think at the time the patch was created it it would have made sense to include a comment as to why those fixes were needed.  But now that I'm removing them I don't see any need to put a comment referencing something that's been removed.  Perhaps I misunderstood?